ceph/init.sls

```diff
@@ -8,7 +8,7 @@ include:
 
 install_ceph_pkgs:
   pkg.installed:
-    - pkgs: {{ ceph.packages }}
+    - pkgs: {{ ceph.packages|tojson }}
```

Comment:
```suggestion
    - pkgs: {{ ceph.packages | json }}
```

@timwsuqld Thanks for this PR.  There is a long history of changes related to this issue and the consistent option we have selected is to use `| json`.  Refer back to https://github.com/saltstack-formulas/consul-formula/pull/40#issuecomment-472728517 and the linked content there for more info.

Reply:
@myii thanks for that. I was just using what was shown in the 2019.2.0 release notes and had forgotten we had `json` as a backwards compatible option.